### PR TITLE
Create helloworld.hs

### PR DESCRIPTION
**Pull request by Priyanka142806**
Created A Hello world program in Haskell

### CODE REVIEW

**Review by C3Suryansu — APPROVED** [review]
Looks good to me.

**Review by trideepb — APPROVED** [review]

**Comment by Hyper-Matrix** [comment]
Great.... maintainers please merge this or change the tag once approved. Thank you for your contribution.

**Pull request merged**
